verifiable-db/src/revelation/revelation_batching.rs

```diff
@@ -0,0 +1,499 @@
+use mp2_common::{types::CBuilder, u256::CircuitBuilderU256, utils::FromTargets, F};
```

Comment:
We can probably remove the `revelation_` from the name since it's already in the module name.

Reply:
Renamed in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/408/commits/b2571453d04e51ebe711251f0a54e2ce7c93ef16